src/logic/basic.lean

```diff
@@ -937,7 +937,7 @@ by simp only [← @forall_eq _ p a, ← forall_and_distrib, ← or_imp_distrib,
 @[simp] theorem forall_eq_or_imp {a' : α} : (∀ a, a = a' ∨ q a → p a) ↔ p a' ∧ ∀ a, q a → p a :=
 by simp only [or_imp_distrib, forall_and_distrib, forall_eq]
 
-@[simp] theorem exists_eq {a' : α} : ∃ a, a = a' := ⟨_, rfl⟩
+theorem exists_eq {a' : α} : ∃ a, a = a' := ⟨_, rfl⟩
```

Comment:
Why does this need removing?

Reply:
Lint says that the new `exists_apply_eq_apply` can prove it. I don't understand why `exists_eq'` is still in simp normal form.

Reply:
Oh right, `exists_apply_eq_apply` was previously in `Type` but is now in `Sort`.